feat(iam): view permission sets (#4072)
scaleway/docs-content: identity-and-access-management/iam/how-to/view-permission-sets.mdx

---
meta:
  title: How to view permission sets
  description: Learn how to view your permission sets in Scaleway IAM.
content:
  h1: How to view permission sets
  paragraph: Learn how to view your permission sets in Scaleway IAM.
tags: iam event-logs
dates:
  validation: 2024-12-03

>   posted: 2023-11-15

Reviewer: ```suggestion
  posted: 2024-12-03
```

---

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization

1. Click **Identity and Access Management (IAM)** from the top-right of the [Organization Dashboard](https://console.scaleway.com/organization) in the Scaleway console. The **Users** tab of the [Identity and Access Management dashboard](https://console.scaleway.com/iam/users) displays.
    <Lightbox src="scaleway-iam-users-dashboard.webp" alt="" />

> 2. Click the name of the user for who you want to view the permissions. The user information page displays.

Reviewer 2: ```suggestion
2. Click the name of the user whose permissions you want to view. The user information page displays.
```

    <Message type="tip">

>     To view your own permissions in someone else's Organization, you can follow the same process and click on your name. If you cannot see your own permissions, contact an admin of the Organization, as you might lack read permissions.

Reviewer: ```suggestion
    To view your own permissions in someone else's Organization, you can follow the same process and click on your name. If you cannot see your own permissions, contact an admin of the Organization, as you may not have the necessary permissions.
```
